packages/nimbus/electron/electron-env.d.ts

```
@@ -1,29 +1,147 @@
+export interface StoredTokenData extends TokenResponse {
+  timestamp: number;
+}
+
+// Error Types
+export interface AuthError extends Error {
+  error_description?: string;
+  error?: string;
+}
+
+// Ensure this is treated as a module
+export {};
```

Biome

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)